simpleflow/swf/process/worker/base.py

```diff
@@ -60,13 +60,11 @@ def __init__(self, domain, task_list, workflow, heartbeat=60,
         # this as "no timeout"
         self._heartbeat = heartbeat or None
 
-        swf.actors.ActivityWorker.__init__(
-            self,
-            domain,
-            task_list,
-            *args,  # directly forward them.
-            **kwargs  # directly forward them.
-        )
+        # Call super()'s methods: as this class uses multiple inheritance, the
```

Comment:
Same remark (and no, args/kwargs are not useful currently; no opinion on keeping/removing these...)